.gitignore

```diff
@@ -0,0 +1 @@
+*.class
```

Comment:
the class filed to not need to be added to the repository since they are generated from the source